src/ctia/http/handler.clj

```diff
@@ -179,6 +185,13 @@
                     :tokenUrl token-url
                     :flow flow}))))
 
+(defn- check-entity-enabled
+  [{{:keys [enabled?]} :FeaturesService}
+   {:keys [entity] :as entity-map}]
+  (if (enabled? entity)
```

Comment:
by using `cond->` one reifies the fact that both the 'if' and 'else' branches relate to the same subject